### DISCUSSION TITLE
Can't access  collaborators because Github web site thinks its sending a code to my phone, which it is not sending

### DISCUSSION THREAD

**Discussion by cboudreaux** [community/community#46483] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am trying to access Collaborators in the Github web site, and the site keep saying that it is sending a code to my mobile app, but the mobile app displays nothing. I am deleting the mobile app from my phone permanently, to avoid this issue in the future.  

I'd like to attach a screen shot, but this form does not appear to have any way to do that.

The idea of forcing everyone to carry their phone everywhere they go, to be able to use the web app makes zero sense. Please stop this terribly bad process.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @cboudreaux ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by cboudreaux** [comment]
Where did you move it?  This is a product issue.

**Reply by queenofcorgis** [reply]
I moved it to our mobile category!

**Comment by cboudreaux** [comment]
It’s not a mobile issue.It’s an issue in the login process on desktops.Anyway,  the person who owns user experience for login needs to address it.No one in the community can fix this.  It’s a product design problem.
